apps/mark-scan/backend/src/custom-paper-handler/state_machine.test.ts

```diff
@@ -1408,6 +1334,8 @@ describe('unrecoverable_error', () => {
   test('triggers when an error occurs during voting_flow.scanning', async () => {
     await executeLoadPaper();
 
+    mockOf(scanAndSave).mockReset();
+    mockOf(interpretSimplexBmdBallot).mockReset();
```

Comment:
There are test assertions in following lines that expect exactly 1 call to these mocks - felt cleaner to reset the mocks than change those expectations